snapcraft/tests/plugins/python/test_pip.py

```diff
@@ -0,0 +1,412 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2017 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import subprocess
+
+import fixtures
+from unittest import mock
+
+from testtools.matchers import (
+    Contains,
+    Equals,
+)
+
+from snapcraft.plugins._python import (
+    _pip,
+    errors,
+)
+
+from snapcraft import (
+    tests
+)
+
+
+def _create_python_binary(base_dir):
```

Comment:
don't you have this same method three times now?
Why not put it in a PythonBaseTestCase class?

Reply:
Ah,  yes of course, good catch.